projects/VitalRecord/VitalRecord_constructors.cs

```diff
@@ -97,6 +97,11 @@ public VitalRecord(string record, bool permissive = false)
             {
                 throw new System.ArgumentException("The given input does not appear to be a valid XML or JSON FHIR record.");
             }
+            // Validate that the given Bundle is not a message.
+            if (this.Bundle.Type == Bundle.BundleType.Message)
+            {
+                throw new BundleTypeException("Expected a Vital Record Bundle, given a Message.");
```

Comment:
Update the message to "The FHIR Bundle must be of type Document, not Message". This matches the error message format we return when we expect a message but received a document